src/modules/utils.js

```diff
@@ -0,0 +1,38 @@
+export const createNewgame=async() => {
+    const New_game = {
+        name: 'Luzinda games',
+      };
+      const key=localStorage.getItem("key");
+      if(key){
+
+        return key
+      }
+   const res = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(New_game),
+      });
+    const data = await res.json();
+    localStorage.setItem("key",data.result.substr(14, 20))
+    return result.substr(14, 20)
+  }
+ export const postNew=async(name,score)=>{
+    const data={
+    "user": name,
+    "score": score
+    }
+    const keys=localStorage.getItem("key");
+    const res = await fetch(`https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${keys}/scores/`, {
+        method: 'POST',
```

Comment:
- [ ] I have noticed that you are fetching the API at first and after submitting a new score, instead of hitting the ```POST API``` endpoint and posting user entries, you are saving the score to the local storage. 😀😀
- [ ] Also when the Refresh button is clicked, instead of hitting the ```GET API``` endpoint causing the scores list change, your app is fetching data for the local storage 

    - As this project is about sending and receiving data from/to the Leaderboard API by using JavaScript async and await, It would be better to avoid using local storage for storing and retrieving scores in a leaderboard API. Instead, it's recommended to use server-side storage and retrieve data through API requests.  😎😎😎😎

    _This is because local storage has limitations such as limited capacity and security concerns, whereas server-side storage provides more robust solutions for storing and retrieving data. Additionally, using API requests allows for better scalability and data management. 🙏_

---
Kindly follow this [link](https://www.geeksforgeeks.org/how-to-use-the-javascript-fetch-api-to-get-data/) or [this video](https://www.youtube.com/watch?v=Yp9KIcSKTNo) for more info on hitting API endpoints

## your project currently ♻️ 
https://www.loom.com/share/ce0e0011ff3a4ac59f3aa46f058a3a9d

Reply:
Hu @TuyishimireEric , I'm not actually fetching data from the local storage, the only thing am fetching from the local storage is the API key which is used to identify my account so as to make the correct API queries for my specific data.

When the refresh button is clicked, the same operation happens ie fetching the API key since the program, kindly provide me some extra comments once you see this.

Reply:
Here is a loom recording of how the project works https://www.loom.com/share/b02e3e1785a8403dbfcff55f3f89189b